mod_reforged/hooks/config/faction_southern.nut

```diff
@@ -1,5 +1,40 @@
 ::MSU.Table.merge(::Const.Tactical.Actor.Conscript, {
-	RangedDefense = 0
+	XP = 200,
+	ActionPoints = 9,
+	Hitpoints = 55,
+	Bravery = 70,
+	Stamina = 120,
+	MeleeSkill = 65,
+	RangedSkill = 50,
+	MeleeDefense = 5,
+	RangedDefense = 0,
+	Initiative = 110,
+	FatigueEffectMult = 1.0,
+	MoraleEffectMult = 1.0,
+	Armor = [
+		0,
+		0
+	],
+	FatigueRecoveryRate = 15
+});
+::MSU.Table.merge(::Const.Tactical.Actor.Gunner, {
+	XP = 250,
+	ActionPoints = 9,
+	Hitpoints = 70,
+	Bravery = 70,
+	Stamina = 120,
+	MeleeSkill = 65,
+	RangedSkill = 75,
+	MeleeDefense = 5,
+	RangedDefense = 20,
```

Comment:
We were previously giving Gunners +10rdef within the entity file because we remove anticipation from their vanilla perks. All this change does is move this to main stats. This change should stay imo.